airflow-core/src/airflow/models/dagrun.py

```diff
@@ -67,9 +67,10 @@
 from airflow.configuration import conf as airflow_conf
 from airflow.exceptions import AirflowException, TaskNotFound
 from airflow.listeners.listener import get_listener_manager
-from airflow.models import Log
+from airflow.models import Deadline, Log
```

Comment:
We shouldn't be adding new "convince" imports to airflow.models (as those were mostly there for users/dag authors in the 2.x era) but import directly from the sub module

Reply:
I let the IDE do it's thing and didn't catch that, thanks.  I'll fix it.